src/loki/isochrone_action.cc

```diff
@@ -64,6 +64,14 @@ void loki_worker_t::init_isochrones(Api& request) {
   parse_costing(request);
 }
 void loki_worker_t::isochrones(Api& request) {
+  // time this whole method and save that statistic
+  midgard::scoped_timer<> t([&request](const midgard::scoped_timer<>::duration_t& elapsed) {
+    auto e = std::chrono::duration_cast<std::chrono::duration<double, std::milli>>(elapsed).count();
+    auto* stat = request.mutable_info()->mutable_statistics()->Add();
+    stat->set_name("loki_worker_t::isochrones");
+    stat->set_value(e);
+  });
```

Comment:
Can we turn this repeated block into a macro?

Reply:
yeah i think we can make a function in worker.h or some other fundamental header and call it something like `make_timer(...)` should be straight forward!